sdk/samples/iot/hub/README.md

```diff
@@ -157,6 +157,10 @@ Start using the IoT Hub Client in your solutions. More SDK details can be found
 
 For extensive documentation on Azure IoT Hub, see the [API reference documentation][iot_hub_mqtt].
```

Comment:
Humm, not your change, but this points to the IoT DPS documentation..not the hub mqtt documentation.

Reply:
can we fix this in your pr?